runtime/shibuya/src/lib.rs

```diff
@@ -1665,6 +1677,52 @@ type EventRecord = frame_system::EventRecord<
     <Runtime as frame_system::Config>::Hash,
 >;
 
+parameter_types! {
+    pub const EnterDuration: BlockNumber = 4 * HOURS;
+    pub const EnterDepositAmount: Balance = 2_000_000 * SBY;
+    pub const ExtendDuration: BlockNumber = 2 * HOURS;
+    pub const ExtendDepositAmount: Balance = 1_000_000 * SBY;
+    pub const ReleaseDelay: u32 = 2 * DAYS;
+}
+
+pub struct SafeModeWhitelistedCalls;
+impl Contains<RuntimeCall> for SafeModeWhitelistedCalls {
+    fn contains(call: &RuntimeCall) -> bool {
+        match call {
+            RuntimeCall::System(_) | RuntimeCall::SafeMode(_) | RuntimeCall::TxPause(_) => true,
+            _ => false,
+        }
+    }
+}
+
+impl pallet_safe_mode::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Currency = Balances;
+    type RuntimeHoldReason = RuntimeHoldReason;
+    type WhitelistedCalls = SafeModeWhitelistedCalls;
+    type EnterDuration = EnterDuration;
+    type EnterDepositAmount = EnterDepositAmount;
+    type ExtendDuration = ExtendDuration;
+    type ExtendDepositAmount = ExtendDepositAmount;
+    type ForceEnterOrigin = EnsureRootWithSuccess<AccountId, ConstU32<9>>;
```

Comment:
A better configuration can be to use a special type for those privileges. You can create an `EmergencyAuthority` and `EmergencyAuthorityWithSuccess` in governance primitives:

```rs
// OpsMaintainer origins

pub type EmergencyAuthority = EitherOfDiverse<MoreThanHalfCouncil, EnsureRootOrAllTechnicalCommittee>;
pub type EmergencyAuthorityWithSuccess<Success = BlockNumberFor<Self>> = EnsureWithSuccess<EmergencyAuthority, AccountId, Success>;
```

and use them in the pallet_safe_mode config like this:

```rs
type ForceEnterOrigin = EmergencyAuthorityWithSuccess<ConstU32<2_400>>; // 2_400 blocks = 4 hours
type ForceExtendOrigin = EmergencyAuthorityWithSuccess<ConstU32<2_400>>; // 2_400 blocks = 4 hours
type ForceExitOrigin = EmergencyAuthority;
type ForceDepositOrigin = EmergencyAuthority;
```